specification/dimensions/dimensions.mdpp

```diff
@@ -15,5 +15,6 @@ dimensions to categorize, filter, and reveal details in your data when grouped w
 !INCLUDE "servicename.md",1
 !INCLUDE "subaccountid.md",1
 !INCLUDE "subaccountname.md",1
+!INCLUDE "billingcurrency.md",1
```

Comment:
Sort in alphabetical order please

Reply:
Done here - https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/pull/117/commits/9277555ed94495826d930061fa20ce0d29aecc13